TitanRobotics2024/src/main/java/frc/robot/Subsystem/DriveBase.java

```diff
@@ -2,71 +2,60 @@
 
 import frc.robot.Data.PortMap;
 
-public class DriveBase implements Subsystem 
-{
+public class DriveBase implements Subsystem {
 
-  @Override
-  public void start() 
-  {
-    // TODO Auto-generated method stub
-    throw new UnsupportedOperationException("Unimplemented method 'start'");
-  }
 
-  public double frontleft;
-  public double frontright;
-  public double backleft;
-  public double backright;
+  private double leftPower;
+  private double rightPower;
 
-  private final ModifiedMotors motorFrontLeft;
-  private final ModifiedMotors motorRearLeft;
-  private final ModifiedMotors motorFrontRight;
-  private final ModifiedMotors motorRearRight;
+  private ModifiedMotors motorRearLeft;
+  private ModifiedMotors motorFrontRight;
+  private ModifiedMotors motorRearRight;
+  private ModifiedMotors motorFrontLeft;
 
   private static DriveBase instance = null;
 
-  public static DriveBase getInstance() 
-  {
+  public static DriveBase getInstance() {
     if (instance == null) {
       instance = new DriveBase();
     }
     return instance;
   }
 
-  public DriveBase() 
-  {
-    motorFrontLeft = new ModifiedMotors(PortMap.FRONTLEFT.portNumber);
-    motorRearLeft = new ModifiedMotors(PortMap.REARLEFT.portNumber);
-    motorFrontRight = new ModifiedMotors(PortMap.FRONTRIGHT.portNumber);
-    motorRearRight = new ModifiedMotors(PortMap.REARRIGHT.portNumber);
+  public DriveBase() {
+    this.motorFrontLeft  =  new ModifiedMotors(PortMap.FRONTLEFT.portNumber, "CANVictorSPX");
+    this.motorRearLeft   =  new ModifiedMotors(PortMap.REARLEFT.portNumber, "CANVictorSPX");
+    this.motorFrontRight =  new ModifiedMotors(PortMap.FRONTRIGHT.portNumber, "CANVictorSPX");
+    this.motorRearRight  =  new ModifiedMotors(PortMap.REARRIGHT.portNumber, "CANVictorSPX");
+
   }
 
-  public void setRightSideMotors(double power) 
-  {
-    // set the right side motors to "power"
+  public void setRightSideMotors(double power) {
+    this.rightPower = power;
   }
 
-  public void setLeftSideMotors(double power) 
-  {
-    // set the left side motors to "power"
+  public void setLeftSideMotors(double power) {
+    this.leftPower = power;
   }
```

Comment:
Prefer refactoring function names to `setRightPower()` shorter and more accurate discription